tests/repository_simulator.py

```diff
@@ -368,9 +369,9 @@ def write(self):
 
         for ver in range(1, len(self.signed_roots) + 1):
             with open(os.path.join(dest_dir, f"{ver}.root.json"), "wb") as f:
-                f.write(self._fetch_metadata("root", ver))
+                f.write(self._fetch_metadata(RoleNames.ROOT.value, ver))
 
-        for role in ["timestamp", "snapshot", "targets"]:
+        for role in [RoleNames.TIMESTAMP.value, RoleNames.SNAPSHOT.value, RoleNames.TARGETS.value]:
```

Comment:
You can make it shorter:
```suggestion
        for role in RoleNames:
```